src/Pages/Practice.js

```diff
@@ -6,7 +6,7 @@ import Paper from '@mui/material/Paper';
 import Grid from '@mui/material/Grid';
 import Stepper from '@mui/material/Stepper';
 import Step from '@mui/material/Step';
-import axios from 'axios';
+import axios, { AxiosHeaders } from 'axios';
```

Comment:
I don't see why was AxiosHeaders imported , as you never used it in this file I guess

Reply:
I guess that happened by mistake while writing axios somewhere, should I remove it?

Reply:
Yes remove it , its the reason for the failed checks.
So the way netlify is been set, it treats warnings as errors, and hence there was a warning saying that axiosHeaders is defined but never used, making the deployment give a failed check. 
You can always check the deploy log given in the comment by netlify bot to see whats wrong. 
Just remove all the warnings and  make a commit to this PR , and we are good to go to merge this one .
Thanks a lot for your contribution :)

Reply:
Oh sorry, the reason you aren't able to see, is that it shows to me as I am signed with my account with which this repo is been deployed. I will try to see if there's a way for contributors to see the log too. 
Also, one more thing to notice is that right now there is repetitive code on Pratice.js and Tags.js. As we almost use the same logic and functions on both components. So A good idea to remove the repetitive code is to use a custom Hook named something like  'useGetSolvedProblems'. 
You can always look about custom hooks on youtube or for some blog on internet.
For now, I will merge this PR, and then you can make a new PR for the above code refactoring, this would also help you to make more PRs for your KWOC contribution.

Reply:
Reply to this, as an affirmation that you have read, so that I can merge and close this PR.